cmd/pbapi/main.go

```diff
@@ -83,7 +85,20 @@ func main() {
 	// initialize cache
 	cache.Initialize()
 
-	// initialize the job queue
+	// initialize platform bus
+	if config.Kafka.Enabled {
+		err = bus.InitializeKafkaBroker()
+		if err != nil {
+			logger.Fatal().Err(err).Msg("Unable to initialize the platform bus")
+		}
+	}
+
+	// initialize background goroutines
+	bgCtx, bgCancel := context.WithCancel(ctx)
+	bg.Initialize(bgCtx)
+	defer bgCancel()
+
```

Comment:
There are two new packages, `bus` represents our platform Kafka bus, I wanted something short for the package name. Also it is a generic interface with three implementations:

* noop broker (does nothing, messages are thrown away and never delivered, this is the default if you don't enable kafka)
* stub broker (in memory message delivery, used in unit tests)
* kafka (well, kafka broker)

The second new package is `bg` because we need REST API to be fast, so all availability checks create messages and send them over a Go channel to a background routine which picks it up and send it in 1k batches (not later than in 5 seconds). Kafka works best with multiple messages, not one by one. This allows the REST API to be super-fast.